packages/sdks/python/tester-local/tests/test_get_doc_query_status.py

```
@@ -0,0 +1,25 @@
+import os
+import time
```

Ruff (0.11.9)

`time` imported but unused

Remove unused import: `time`

(F401)